spec/tests/PlayerEvents.spec.ts

```diff
@@ -248,18 +248,26 @@ describe('player event tests', () => {
     });
   });
 
-  describe('bitrate tracking', () => {
+  describe('video quality tracking', () => {
     it('report bitrate on event', () => {
       playerMock.eventEmitter.firePlayEvent();
       playerMock.eventEmitter.firePlayingEvent();
-      playerMock.eventEmitter.fireVideoPlaybackQualityChangedEvent(2_400_000);
+      playerMock.eventEmitter.fireVideoPlaybackQualityChangedEvent(2_400_000, 30);
```

Comment:
Please add a test case where we don't pass a frame rate and explicitly check that we don't set it (so we don't accidentally break it in the future)